src/pages/GroupInvitePage.tsx

```diff
@@ -24,27 +25,36 @@ const animationOpts = {
   ease: [0.16, 1, 0.3, 1],
 };
 
-function GroupInvitePage({ inviteCode }) {
+type GroupInvitePageProps = {
+  inviteCode: string;
+};
+
+function GroupInvitePage({ inviteCode } : GroupInvitePageProps): JSX.Element {
   const [accepted, setAccepted] = useState(false);
-  const [group, setGroup] = useState(null);
+  const [group, setGroup] = useState<Group | null>(null);
   const [isLoadingGroup, setLoadingGroup] = useState(false);
   const [isAcceptingInvite, setAcceptingInvite] = useState(false);
-  const [error, setError] = useState(null);
+  const [error, setError] = useState<Error | null>(null);
   const [isLinkInvalid, setIsLinkInvalid] = useState(false);
 
   const history = useHistory();
   const params = new URLSearchParams(window.location.search);
   const userId = localStorage.getItem('id');
   const groupId = params.get('groupId');
 
-  const getMemberText = numUsers => {
+  const getMemberText = (numUsers: number) => {
     if (numUsers === 1) {
       return `${numUsers.toLocaleString()} member`;
     }
 
     return `${numUsers.toLocaleString()} members`;
   };
 
+  type Error = {
```

Comment:
Might want to use a different name for this type so that it doesn't conflict with the builtin type